packages/dnb-eufemia/src/components/input-masked/hooks/useMultiInputValues.ts

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react'
+import {
+  MultiInputMaskProps,
+  MultiInputMaskValue,
+} from '../MultiInputMask'
+
+type SteppedValuesHook<T extends string> = {
+  inputs: MultiInputMaskProps<T>['inputs']
+  defaultValues?: MultiInputMaskProps<T>['values']
+  callback?: (values: MultiInputMaskProps<T>['values']) => void
+}
+
+export function useMultiInputValue<T extends string>({
+  inputs,
+  defaultValues,
+  callback,
+}: SteppedValuesHook<T>) {
+  const [values, setValues] = useState<MultiInputMaskValue<T>>(
+    defaultValues ?? createDefaultValues()
```

Comment:
```suggestion
    defaultValues ? defaultValues : createDefaultValues()
```

I had a case when ?? was not supported because of an outdated Storybook version in the mono-repo.
So when e.g. the DatePicker will use this, we probably should not use it as of now.